### ISSUE TITLE
Verbose logging

### ISSUE THREAD

**Issue by QuantumInformation** [AnomalyInnovations/aws-api-gateway-cli-test#4]
Is there any way to get more verbose logging, finding it hard to debug this error:

![image](https://user-images.githubusercontent.com/216566/30967191-540a2b5e-a453-11e7-9d92-b791a7925e09.png)

**Comment by QuantumInformation** [comment]
Eventually, I had to update the trust relationship.

**Comment by jayair** [comment]
@QuantumInformation Yeah I need to look into how we can get more info from AWS. Their error messages lack quite a bit of detail.

The best we've been able to do so far is just look at what people experience when they see a certain error message and relay that to everybody else.

**Comment by lopezdp** [comment]
Im getting a status: 401 'Unauthorized' response message using the same data but configured for my credentials.... any suggestions on what path to take to debug this? thanks for the help!

**Comment by jayair** [comment]
Hey @lopezdp, is this related to the issue you posted on the forums?

**Comment by lopezdp** [comment]
> Hey @lopezdp, is this related to the issue you posted on the forums?

This was something else but I resolved it. I had the wrong info down for the authorizer attribute in serverless.yml. Thanks again!
